docs/usage.md

```diff
@@ -361,4 +361,3 @@ For example, it will not tell the Python handler to look for packages in these p
 (the paths are not added to the `PYTHONPATH` variable).
 If you want to tell Python where to look for packages and modules,
 see [Python Handler: Finding modules](https://mkdocstrings.github.io/python/usage/#finding-modules).
-
```

Comment:
Was there a single newline at the end, or two? If a single, please leave it.

Reply:
There were 2, you can see it here (on `master`):

- [usage.md](https://github.com/mkdocstrings/mkdocstrings/blob/8c952c9f1202b82c2324cf3cb122eb8f7c57fb86/docs/usage.md?plain=1#L364)
- [troubleshooting.md (some other md file, properly formatted)](https://github.com/mkdocstrings/mkdocstrings/blob/master/docs/troubleshooting.md?plain=1#L232)